.github/workflows/build.yaml

```diff
@@ -28,6 +28,14 @@ jobs:
       build_type: branch
       script: "ci/build_wheel.sh"
       matrix_filter: ${{ needs.compute-matrix.outputs.BUILD_MATRIX }}
+  build-conda:
+    needs:
+      - compute-matrix
+    uses: rapidsai/shared-workflows/.github/workflows/conda-python-build.yaml@branch-24.02
+    with:
+      build_type: pull-request
```

Comment:
this should be `build_type: ${{ inputs.build_type || 'branch' }}`.

also, it will need the following:

```yaml
branch: ${{ inputs.branch }}
date: ${{ inputs.date }}
sha: ${{ inputs.sha }}
```

Reply:
This is the `build-conda` portion, which IIUC is just for testing the packages do build

https://github.com/rapidsai/pynvjitlink/blob/a1479d31c23120b65532e8f31f20b64a69e19d5c/.github/workflows/build.yaml#L31-L38

AFAICT this matches what the `build-wheels` section has

https://github.com/rapidsai/pynvjitlink/blob/a1479d31c23120b65532e8f31f20b64a69e19d5c/.github/workflows/build.yaml#L23-L30

<hr>

The `publish-conda` section has these changes already

https://github.com/rapidsai/pynvjitlink/blob/a1479d31c23120b65532e8f31f20b64a69e19d5c/.github/workflows/build.yaml#L50-L59

It appears similar to `publish-wheels`, which also has them

https://github.com/rapidsai/pynvjitlink/blob/a1479d31c23120b65532e8f31f20b64a69e19d5c/.github/workflows/build.yaml#L39-L49

<hr>

Based on this, do we have what we need already?

Or do both Conda & Wheels sections need some updates?

Reply:
That said, it is odd that `build-conda` has `build_type: pull-request` instead of `build_type: branch` like in `build-wheels`. Fixing with PR ( https://github.com/rapidsai/pynvjitlink/pull/43 )

Reply:
It looks like the `build_type` change noted above was all that was needed. Though please let us know if we are missing something